challenge.rb

```diff
@@ -67,6 +93,8 @@ def ac_monitor current_temp, ac_working, desired_temp
 # and print out the result of `fizz_buzz_calculator` for that number.
 # This question will be graded independently of question four.
 
-# Replace this comment with your code.
-
-
+def fizz_buzz(min, max)
+  (min..max).each do |num|
+    fizz_buzz_calculator(num)
+  end
```

Comment:
So close! This time you were asked to print out the results - it should have been 
`puts fizz_buzz_calculator(num)`